### PR TITLE
DNS: URI records: bump python-dns requirements

### PR DESCRIPTION
**Pull request by mbasti-rh**
Support for DNS URI records has been added in python-dns 1.13

https://fedorahosted.org/freeipa/ticket/6344

### CODE REVIEW

**Comment by tiran** [comment]
You forgot to bump ```ipasetup.py.in```.

File: freeipa.spec.in [hunk 1]
```diff
@@ -125,7 +125,7 @@ BuildRequires:  python-lxml
 # 5.0.0: QRCode.print_ascii
 BuildRequires:  python-qrcode-core >= 5.0.0
 # 1.11.0: resolver.YXDOMAIN, Resolver.set_flags
```
Comment (tiran) [inline freeipa.spec.in]: Sorry, I didn't see the comment before. Please update or remove the outdated comment.

**Comment by mbasti-rh** [comment]
Thank you, fixed.

**Comment by mbasti-rh** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/a291c6ded91611ea2bd1a1fdb96314721d73a75f

**Pull request closed without merging**